invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LoanDetails/state/__tests__/reducer.js

```diff
@@ -38,51 +38,8 @@ describe('Fetch loan details reducer', () => {
     expect(reducer(initialState, action)).toEqual({
       ...initialState,
       isLoading: false,
-      data: 'Error',
+      error: 'Error',
       hasError: true,
     });
   });
 });
-
```

Comment:
why do we remove all of those tests?

Reply:
I mistakenly did that, actually they shouldn't be removed just change the types from `ACTION_*` to `IS_LOADING`, `HAS_ERROR` etc..

Reply:
All `ACTION_*` types were removed in favor of using `IS_LOADING`, `SUCCESS`, `HAS_ERROR` since we saw no need to differentiate between those types anymore.

Reply:
Those tests should be removed, right? There are already identical tests for `IS_LOADING`, `HAS_ERROR` etc.